Maple2.Server.Game/Scripting/Npc/NpcScriptContext.cs

```diff
@@ -255,4 +255,29 @@ public class NpcScriptContext {
     public bool HasMesos(long amount) {
         return Session.Currency.Meso >= amount;
     }
+
+    public int Job() {
+        return (int)Session.Player.Value.Character.Job;
+    }
+
+    public int Level() {
+        return Session.Player.Value.Character.Level;
+    }
+
+    /* TODO: Quests aren't implemented yet :( Finish once they are, I guess.
+    public bool QuestState(int questId, int questState) {
+        return Session.Quest.Find(questId).State == questState;
+
+    public bool MultiQuestState(List<int> questIds, int questState) {
+        foreach (int questId in questIds) {
+            if (Session.Quest.Find(questId).State == questState) {
+                return true;
+            }
+        }
+    }
+    } */
+
+    public bool CurrentMap(int mapId) {
```

Comment:
Seems more flexible to not pass `mapId` here, just return `Session.Field.MapId` and you can check against `mapId` in the script.